app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
+    config[:account_suspension]
+  end
+
+  def load_account_inactivity
+    config[:account_inactivity]
+  end
+
+  def load_contract_unpaid_time
+    config[:contract_unpaid_time]
+  end
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid_configuration?
+    config && @valid_configuration
```

Comment:
Rename to `valid?` and implement what is suggested in https://github.com/3scale/porta/pull/574/files#r254209038